### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by timea-techgirl** [pirate-505/tgcleanup#1]
In my telegram chat export, I have all the groups I ever joined, I have no messages in the groups except "xxx joined". I'm unable to delete these messages directly.
Can I remove these joined messages from groups even if I'm not the admin with your tool?

**Comment by pirate-505** [comment]
No, it seems that these messages can not be deleted from the client side in any way.

**Comment by timea-techgirl** [comment]
So, they will appear in the chat export forever right?
